libopenage/renderer/opengl/window.cpp

```diff
@@ -57,6 +57,23 @@ GlWindow::GlWindow(const std::string &title,
 	this->window->setFormat(format);
 	this->window->create();
 
+	// set display mode
+	// Reset to a known state
+	this->window->setWindowState(Qt::WindowNoState);
+	switch (settings.mode) {
+	case window_mode::WINDOWED:
+		this->window->setFlags(this->window->flags() & ~Qt::FramelessWindowHint);
```

Comment:
I think you don't even need this line? Not having the `Qt::FramelessWindowHint` is the default I think.